owner/manager.go

```diff
@@ -223,11 +224,11 @@ func (m *ownerManager) campaignLoop(ctx context.Context, etcdSession *concurrenc
 
 		select {
 		case <-etcdSession.Done():
-			log.Infof("%s etcd session is done, creates a new one", logPrefix)
+			logutil.Logger(context.Background()).Info("etcd session is done, creates a new one", zap.String("logPrefix", logPrefix))
```

Comment:
Could we use `log.WithKeyValue` to replace `context.Background()` in this file.
The key is `logPrefix`.

Reply:
I think it is not necessary to replace `log.WithKeyValue` with `context.Background()`.
Because others are using  `context.Background()`, use it will keep consistent with others.
Using `log.WithKeyValue` will reduce a parameter(`logprefix`), however, I think `context.Background()` is enough.
`log.WithKeyValue` is not better than `context.Background()`.

Reply:
It will eventually be replaced and it can be replaced here without affecting other places, so I think it can be done directly here.

Reply:
done